src/Mod/Draft/draftmake/make_polararray.py

```diff
@@ -60,6 +61,29 @@ def make_polar_array(base_object,
         It defaults to the origin `App.Vector(0, 0, 0)`.
         The vector indicating the center of rotation of the array.
 
+    axis_object: str or DocumentObject, optional
+        It defaults to `None`.
+        This parameter should be the name of an `DocumentObject` or
+        the `DocumentObject` itself.
```

Comment:
Here I would clarify that the document object has to be a `Part::Feature`, because these have `Shapes`. A document object is also an `App::FeaturePython`, or a Group, and these don't have features.